Helpers.gs

```diff
@@ -636,12 +636,12 @@ function processEventInstance(recEvent){
  * Deletes all events from the target calendar that no longer exist in the source calendars.
  * If onlyFutureEvents is set to true, events that have taken place since the last sync are also removed.
  */
-function processEventCleanup(){
+function processEventCleanup(cleaupPastEvents){
```

Comment:
I don't think you need to pass this through as an argument to the method. I think you can just reference this setting variable in line 644 without this argument.

Reply:
You're right, I've also found that it doesn't keep events properly that have passed on the current day, just a minuite